sdk/storage/azure-storage-queues/src/queue_responses.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/storage/queues/queue_responses.hpp"
+
+#include "azure/storage/queues/queue_service_client.hpp"
+
+namespace Azure { namespace Storage { namespace Queues {
+
+  void ListQueuesPagedResponse::OnNextPage(const Azure::Core::Context& context)
+  {
+    m_operationOptions.ContinuationToken = NextPageToken;
+    *this = m_queueServiceClient->ListQueues(m_operationOptions, context);
```

Comment:
the `CurrentPageToken` is missing to be updated

Reply:
`CurrentPageToken` and `NextPageToken` are updated in the call of `ListQueues()`